### ISSUE TITLE
there are too many code in /example are using: from asyncio import *

### ISSUE THREAD

**Issue by hanbeibei** [python/asyncio#464]
does this adapt to pep8?

I was told not to write code in this way. this will add more work to the people who want to use your work, we are not writing perl.

**Comment by gvanrossum** [comment]
You're right, the examples should be changed to just import what they need.

Do you want to help by submitting a PR?

**Comment by denisra** [comment]
I'm willing to work on this one if nobody else is doing it yet.
